test/uk/gov/hmrc/eusubsidycompliancefrontend/controllers/FinancialDashboardControllerSpec.scala

```diff
@@ -64,6 +69,13 @@ class FinancialDashboardControllerSpec
       "play.filters.csp.nonce.enabled" -> false
     )
   )
+  override lazy val appConfig: AppConfig = app.injector.instanceOf[AppConfig]
```

Comment:
how comes these are now needed? whats different in the new test?

Reply:
removed - not needed - copied from an example